integration_tests/sdk/aqueduct_tests/operator_test.py

```diff
@@ -0,0 +1,44 @@
+from aqueduct.decorator import to_operator
+
+from aqueduct import op
+from sdk.aqueduct_tests.test_function import dummy_sentiment_model_function
+
+from ..shared.data_objects import DataObject
+from ..shared.utils import extract
+
+
+def test_to_operator_local_function(client, data_integration):
+    table_artifact = extract(data_integration, DataObject.SENTIMENT)
+
+    @op
+    def dummy_sentiment_model(df):
+        df["positivity"] = 123
+        return df
+
+    def dummy_sentiment_model_func(df):
+        df["positivity"] = 123
+        return df
+
+    output_artifact_from_decorator = dummy_sentiment_model(table_artifact)
+    df_normal = output_artifact_from_decorator.get()
+    output_artifact_from_to_operator = to_operator(dummy_sentiment_model_func)(table_artifact)
+    df_func = output_artifact_from_to_operator.get()
+
+    assert df_normal["positivity"].equals(df_func["positivity"])
+
+
```

Comment:
For some reason, moving the tests down one and adding __init__.py to both the parent and child directories causes this test to fail (but everything still works outside of pytest). Any idea what's happening here? I put some more information in the Linear task itself.

Reply:
could it be the line `from sdk.aqueduct_tests. ...`? It seems other files use `from .test_functions ...` instead

Reply:
yep I tried that do to no avail